### PR TITLE
New Camp View with extra fields added(gis_url, etc),corrected PR review

### PR DESCRIPTION
**Pull request by souravsarangi**
...iew

### CODE REVIEW

**Comment by souravsarangi** [comment]
I am working on editing part. Will do it in a new PR

File: app/views/camp/view.php [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+                                        <b>Camp Name: " . $list['name'] . "</b>
+                                    
+                                        <div class='btn btn-info show' style='float: right;'>Show More</div>
+                                      </div>
+                                      ";
+                                echo "<div class='panel-body' id='panelBody1' style='display: none;'>";
+                                echo "<ul class='list-group'>";
+                                echo "<li class='list-group-item'><b> Organisation Name: </b>" . $list['organisation_name'] . " </li><li class='list-group-item'><b>Camp_Head:  </b>".$list['fname']." ".$list['lname']."</li><li class='list-group-item'><b>Contact Number: </b>".$list['phone_no']."</li><li class='list-group-item'><b>Email:  </b>".$list['email']."</li>"."</li><li class='list-group-item'><b>Mailing List:  </b>".$list['mailing_list']."</li>"."</li><li class='list-group-item'><b>Mailing Address:  </b>".$list['mailing_address']."</li>"."</li><li class='list-group-item'><b>Gis URL:  </b>".$list['gis_url']."</li>";
```
Comment (gauravmittal1995) [inline app/views/camp/view.php:125]: Now that GIS is implemented, i think we are going to remove gis_url

**Comment by gauravmittal1995** [comment]
Please check the title of PR before submitting, Should not overflow to description. Please Change this.

File: app/views/camp/view.php (hunk 1 above)
Comment (nisargjhaveri) [inline app/views/camp/view.php:125]: It is still not merged, so either we'll have to put this on hold or merge it change later.

Comment (souravsarangi) [inline app/views/camp/view.php:125]: okay let me know when you are changing the schema

**Comment by gauravmittal1995** [comment]
Closing

**Pull request closed without merging**